lua/packer/compile.lua

```diff
@@ -637,9 +637,9 @@ local function make_loaders(_, plugins, should_profile)
   if next(ftdetect_paths) then
     table.insert(result, 'vim.cmd [[augroup filetypedetect]]')
     for _, path in ipairs(ftdetect_paths) do
-      local escaped_path = vim.fn.escape(path, ' ')
+      local escaped_path = vim.fn.escape(path, ' \\')
```

Comment:
How will this impact non windows machines OOI @EdenEast? haven't tried this myself as I currently don't have the time.

Reply:
I tested these changes on linux as well and there was no issues. As I went to bed I realized that another solution is to use `[[ ]]` instead of quotes to make the `\` character not be treated as an escape character. I like this solution better for this issue.